bundle-workflow/src/read_write_s3.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import boto3
+
+
+class ReadWriteFiles:
+
+    # get_S3Objects:Used for getting the files from S3
+    # bucket_name: Name of the bucket you want to get the file from
+    # role_arn: Amazon Resource Name (ARN) of the role that you want to assume
+    # role_session_name: Name for the session
+    # bucket_bundle: Object path from where you want to get the bundle
+    # bucket_manifest: Object path from where you want to get the manifest
+    # path_to_save_bundle: Path where you want to save the bundle at
+    # path_to_save_manifest: Path where you want to save the manifest at
+    def __init__(self, role_arn, role_session_name):
+        self.role_arn = role_arn
+        self.role_session_name = role_session_name
+
+    def get_S3_objects(self, bucket_name, bucket_bundle, bucket_manifest, path_to_save_bundle, path_to_save_manifest):
```

Comment:
Is getting the manifest mandatory for this method? What if I just have an S3 path and want to get the files that are in that bucket, for example for DependencyInstaller. Currently the code here is assuming that all the user passes in all the parameters.

Reply:
I have modified the method to take only the bundle parameter where through the S3 path we can get the files that are in that bucket